examples/nats-basic-client/publisher.bal

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We do not include license headers for examples since they will be displayed on the website.